api/HTMLElement.json

```diff
@@ -1075,25 +1075,24 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": {
-              "version_added": "79",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.forms.enterkeyhint",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox": [
+              {
+                "version_added": "94"
+              },
+              {
+                "version_added": "79",
+                "version_removed": "94",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.forms.enterkeyhint",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+            ],
             "firefox_android": {
-              "version_added": "79",
```

Comment:
Note, I removed this because FF android from v79 does not have method to set that preference - as discussed in #9679